fetcher.py

```diff
@@ -10,7 +10,17 @@ def version(self):
     return parse_version(json.loads(self.open('https://mtgjson.com/json/version.json')))
 
   def all_cards(self):
-    return json.loads(self.open('https://mtgjson.com/json/AllCards.json'))
+    if os.path.isdir('./ziptemp'):
```

Comment:
We let the user choose locations for db and images. I wonder if we should just have single entry in config.json for "directory to store stuff in" and put this in it too.

Then we can not delete it afterwards and check its timestamp before bothering to re-download it (if it's less than hour old, say).